src/content/url-handling.js

```diff
@@ -50,7 +50,10 @@ export function urlFromState(urlState) {
     invertCallstack: urlState.invertCallstack ? null : undefined,
     jsOnly: urlState.jsOnly ? null : undefined,
     callTreeFilters: stringifyCallTreeFilters(urlState.callTreeFilters[urlState.selectedThread]) || undefined,
-  } : {});
+  } : (urlState.selectedTab === 'flameChart' ? {
```

Comment:
Oh and also, I think it would be good for code legibility to have an eslint rule disabling nested ternaries. `</bikeshed>`